app/controllers/claims_controller.rb

```diff
@@ -156,6 +178,10 @@ def set_backlink_path
     @backlink_path = claim_path(current_journey_routing_name, previous_slug) if previous_slug.present?
   end
 
+  def set_any_backlink_override
+    @backlink_path = @form.backlink_path if @form.backlink_path
```

Comment:
The banklink path will likely be different for different journeys, so this highlights the need to be able to override forms on a per-journey basis.

Reply:
OK thanks, sorry the method above wasn't visible in the diff.